views/index.html.twig

```diff
@@ -34,8 +34,8 @@
             {% if name %}{{ name }}{% else %}Composer repository{% endif %}
         </h1>
 
-        <small id="lastupdate">
-            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::ISO8601')) }}">{{ "now"|date(constant('\DateTime::RSS')) }}</time>
+        <small>
+            Last updated: <time datetime="{{ "now"|date(constant('\DateTime::RFC3339')) }}">{{ "now"|date(constant('\DateTime::RFC3339')) }}</time>
```

Comment:
you are using RFC3339 for both the machine-readable representation (fine) and the human-readable one (less fine) here. There is no Date_COOKIE involved here